book/IMaEA.md

```diff
@@ -4499,7 +4499,7 @@ $
 
 A common pattern in the Exiv2 code is the table/function pattern. 
 
-| Fuction         | Purpose | 
+| Function        | Purpose |
```

Comment:
This was the only instance where I harmonized whitespace, I also trimmed the trailing space while I was here.